src/components/ui/Keybinding.tsx

```
@@ -33,10 +34,18 @@ export function Keybinding({ command, combo: comboProvided }: KeybindingProps) {
         return k;
     }
   };
-  const bindings = useSettingsStore(
-    state => state.getSetting("keybindings") as Record<string, string[]>
-  );
-  const comboStr = comboProvided ?? (command ? bindings[command]?.[0] : "") ?? "";
+  
+  let bindings: Record<string, string[]> | undefined;
+  try {
+    bindings = useSettingsStore(
```

ESLint

[error] React Hook "useSettingsStore" is called conditionally. React Hooks must be called in the exact same order in every component render.

(react-hooks/rules-of-hooks)